Automattic/wp-calypso
# Themes: remove the last unneeded part of universal themes

Most Universal Themes functionality was removed in #65648 but we left support for removing the Nav Menus item for Block Themes.

But, the Nav Menus menu item is [only added in Core](https://github.com/WordPress/WordPress/blob/812b1e296c57c53c6a2bf23f2cbc62adf4c7cc23/wp-admin/menu.php#L225-L227) when `add_theme_support` is called for `'menus'` or `'widgets'`, which would not be the case for a block theme in any case, so `hide_nav_menus_submenu` is effectively a no-op.

## Proposed Changes

* Remove deprecated `wpcom-universal-themes` functionality. The FSE beta is long over, long live FSE!

## Testing Instructions

With this applied to your wpcom sandbox, try activating FSE themes and non-FSE themes. You should see the presence or absence of Appearance -> Nav Menus unaffected by this Diff in either case. I found no evidence of any block theme that registers menus or widgets so this would never have had effect anyway.

## Pre-merge Checklist
